addons/sale_project/static/tests/tours/task_create_sol_tour.js

```diff
@@ -0,0 +1,39 @@
+import { registry } from "@web/core/registry";
+import { stepUtils } from "@web_tour/tour_service/tour_utils";
+
+registry.category("web_tour.tours").add("task_create_sol_tour", {
+    test: true,
+    url: "/web",
+    steps: () => [stepUtils.showAppsMenuItem(),
+{
```

Comment:
Same as before regarding indentation

Reply:
Hello, @pkyriakou I've done suggested changes let me know if anything needs to be chnaged.